src/DetailsView/components/issue-filing-dialog.tsx

```diff
@@ -121,7 +121,7 @@ export class IssueFilingDialog extends React.Component<
         this.props.deps.issueFilingActionMessageCreator.fileIssue(
             ev,
             service,
-            this.props.selectedIssueData,
+            this.props.selectedIssueData!,
```

Comment:
It should always be defined if the dialog is open, and since this is the primary button's onClick callback we know that the dialog will be open when this is called. It's an optional property because it may be undefined if the dialog is closed (i.e. before any issue is selected)